app/helpers/admin/categories_helper.rb

```diff
@@ -2,4 +2,9 @@ module Admin::CategoriesHelper
   def load_sub_category category
     @sub_cats = Category.sub_cats(category.id)
   end
+
+  def check_parent_category category
+    return true if category.parent_id.nil? || category.parent_id != 0
+    false
```

Comment:
đâu cần line này chú `false` chú nhỉ

ngoài ra nếu chú làm cho clear sao ko viết trên toán tử ngôi số 3
`(category.parent_id.nil? || category.parent_id != 0) ? true : false`